### PR TITLE
Display merchants on merchant dashboard

### PR DESCRIPTION
**Pull request by burieberry**

### CODE REVIEW

File: packages/web-client/app/resources/merchant-info.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  private async fetchMerchantInfo(
+    infoDID: string,
+    waitForInfo = false
+  ): Promise<void> {
+    let did = await this.#didResolver.resolve(infoDID);
+    let alsoKnownAs = did?.didDocument?.alsoKnownAs;
+
+    if (alsoKnownAs) {
+      let jsonApiDocument = await this.fetchJson(alsoKnownAs[0], waitForInfo);
+      console.log('json', jsonApiDocument);
```
Comment (backspace) [inline packages/web-client/app/resources/merchant-info.ts:63]: I’m guessing this is from iteration in development and could be removed?

File: packages/web-client/app/templates/card-pay/balances.hbs [hunk 2]
```diff
@@ -1,81 +1,74 @@
...
+              <CardPay::PrepaidCardSafe @safe={{prepaidCard}} />
+            {{/each}}
           </section>
-        {{/if}}
-      </aside>
+        </section>
+      </section>
     {{/if}}
   </section>
-</section>
```
Comment (backspace) [inline packages/web-client/app/templates/card-pay/balances.hbs:71]: I’m not sure what these changes are about, maybe just rearrangements due to CSS?

**Review by backspace — APPROVED** [review]
There’s a likely-not-meant-to-be-included `console.log` but this looks good to me. I think if we have another repetition of the DID-resolving/`fetchJson` pattern we could extract some common functions for resources, but for now the duplication seems fine.

File: packages/web-client/app/resources/merchant-info.ts (hunk 1 above)
Comment (lukemelia) [reply packages/web-client/app/resources/merchant-info.ts:63]: @backspace good catch. Please push a commit to remove this and then merge this PR. :shipit:

**Pull request merged**